nginx/nginx.conf.dev

```diff
@@ -0,0 +1,21 @@
+events {
+}
+
+http {
+    sendfile on;
+
+    server {
+        listen 80;
+        root /www/data/;
+        access_log /var/log/nginx/access.log;
+
+        add_header X-Content-Type-Options    "nosniff" always;
+        add_header Referrer-Policy           "no-referrer-when-downgrade" always;
+        add_header Permissions-Policy        "interest-cohort=()" always;
+        add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
+
+        location /api/ {
+            proxy_pass http://api:8000/api/;
```

Comment:
```suggestion
            proxy_pass remote_backend_server_address;
```

Reply:
for this separate env file is needed which i will add later , this nginx setup is very basic. Its TODO work i didn't specify there as it is a config file.